### ISSUE TITLE
TypeError: (0 , _reactFastCompare.default) is not a function when running tests  

### ISSUE THREAD

**Issue by ah-aberg** [popperjs/react-popper#345]
Get the following error when running tests with V2.2

TypeError: (0 , _reactFastCompare.default) is not a function
11:34:40        node_modules/react-popper/lib/cjs/usePopper.js:75:42

**Comment by FezVrasta** [comment]
Hi, do you have all the dependencies installed? `react-fast-compare` is a new dependency

**Comment by ah-aberg** [comment]
But it isn't a peer dependency, I shouldn't have to install it... that should just happen.
It looks like some other package uses an older version, so possible conflict?

**Comment by FezVrasta** [comment]
I'm not sure how to help without a reproduction example I can play with, sorry.

**Comment by mliq** [comment]
I have the same issue and a clean install does not fix it. I have gone back to 2.1.

Thank you for your hard work on this, but please be more considerate of following semver, as this can cause quite an issue with many people's applications and cost a lot of time in a big team, as it did for us today.

The Release notes for 2.2 say:
> This release is shipped as a minor patch but it's actually a complete rewrite of the library

2 things wrong with this:

1. Anything that is a 'complete rewrite' is clearly going to break something somewhere and should be a major version.

2. It was released as a Minor version, _not_ a "patch". Patch means something very specific in semver.

reference: https://semver.org/

**Comment by mliq** [comment]
Sorry that I can't make a repro at the moment, but it may have to do with `@storybook/components`, which depends on `"react-popper-tooltip": "^2.8.3",`, which depends on `"react-popper": "^1.3.7"`

**Comment by atomiks** [comment]
`react-fast-compare` only ships a CJS version, but Popper imports it as a module. Usually this is fine, but not in the case of Jest, because of the following I think:

```js
// this
import isEqual from 'react-fast-compare';

// becomes
const isEqual = require('react-fast-compare').default;

// but the function is not at `.default`.
```

 I think bundlers usually don't require this with their ESM-CJS interop, but Jest does require it.

**Comment by FezVrasta** [comment]
Why does it work in our tests and in my CRA app then?

**Comment by atomiks** [comment]
It likely depends on the configuration

**Comment by FezVrasta** [comment]
The library does `module.exports = function isEqual(...`, so there's not a named export, it should fallback to the default export.

If I try to do `import { isEqual } ` it throws an error because it's not exported  that way.

**Comment by atomiks** [comment]
https://unpkg.com/browse/react-popper@2.2.0/lib/cjs/usePopper.js check line 75

**Comment by FezVrasta** [comment]
Oh cool, why 😳

**Comment by FezVrasta** [comment]
Fixed in 2.2.1
